setup/scripts/build_docker.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
Thanks for explaining `setup/scripts` and `setup/installer/scripts`.  Do you want to clarify these, such as:
`setup/gcp`, so that you can have `setup/gcp/installer.py` and `setup/gcp/util/`
`setup/build`, and just put all the scripts there.  Do you think that we can drop the `scripts` as a directory?

Reply:
I think we should have a separation of scripts used on GCP deployment from those used to build and test on Travis. But I agree with the suggestion on renaming `setup/installer/installer_runner.py` to `setup/gcp/installer.py`